assets/src/scripts/components/hydrograph/timeSeries.js

```diff
@@ -136,3 +137,18 @@ export const descriptionSelector = createSelector(
         }
     }
 );
+
+export const tsTimeZoneSelector = createSelector(
+    getIanaTimeZone,
+    getNwisTimeZone,
+    (ianaTimeZone, nwisTimeZone) => {
+        const tz = DateTime.fromObject({zone: ianaTimeZone});
+        const tzShortName = tz.offsetNameShort.toUpperCase();
+        const nwisShortNames = Object.values(nwisTimeZone).map(tz => tz.zoneAbbreviation.toUpperCase());
+        if (nwisShortNames.includes(tzShortName)) {
+            return ianaTimeZone;
+        } else {
+            return 'UTC';
```

Comment:
I'm on the fence about whether using UTC as backup timezone would be confusing... :thinking:

Reply:
I do think it should return local time zone if ianaTimeZone is null.

Reply:
The only site that I've found so far is 251549080251200, which the weather service reports as being in UTC, but NWISWeb reports as being in eastern time.

Reply:
The reason I thought local would be better is because NWIS will typically report two timezones (ST and DT). Which to use?

Reply:
So from the NWIS time zones (e.g. EST, MST, etc) and NWIS code about whether or not a site observes daylight savings, I think we could potentially infer the correct IANA code. Then we could let luxon figure out when to use ST or DT.

My main concern would be the reliability of inferring the timezone. It seems more predictable to fall back to local rather than to attempt the inference.

Reply:
Mainly thinking about situations such as Arizona or those parts of Indiana (prior to 2006) that don't observe DT. Things might get messy real quick if we attempt the inference.

Reply:
Yeah that makes sense